### PR TITLE
docs: add @Kamari93 as a contributor

### PR DESCRIPTION
**Pull request by Kamari93**
## Description

Add Kamari93 to the guestbook.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -694,6 +698,12 @@
         "userTesting"
       ]
     },
+    {
```
Comment (BekahHW) [inline .all-contributorsrc:701]: I'm not sure what's going on here, but Christine is now added in multiple times.

File: README.md [hunk 2]
```diff
@@ -123,7 +123,9 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
```
Comment (BekahHW) [inline README.md:126]: There should not be any changes to the spacing in this file.

**Review by BekahHW — CHANGES_REQUESTED** [review]

File: .all-contributorsrc (hunk 1 above)
Comment (Kamari93) [reply .all-contributorsrc]: Hi Bekah I believe I resolved the requested change to the .all-contributorsrc file. Let me know if there are anymore changes you need from me. Thank you!

File: README.md (hunk 2 above)
Comment (Kamari93) [reply README.md]: Hi Bekah I believe I resolved the requested change to the README.md file. Let me know if there are anymore changes you need from me. Thank you!

**Review by adiati98 — APPROVED** [review]
Hi @Kamari93,
We've resolved the merge conflicts and merged your PR! 🎉

Check out [this section about how to resolve conflicts](https://github.com/open-sauced/guestbook#keeping-branch-up-to-date-and-resolving-merge-conflicts) for your future reference.

Also, in the future, please complete the PR form and describe the changes clearly. Doing so will help maintainers to review and understand the changes in your PR, and will it benefit you as well. 🙂

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕

pizza GIF

**Pull request merged**